e2e/install_test.go

```diff
@@ -1401,33 +1401,12 @@ func TestMultiNodeAirgapUpgradePreviousStable(t *testing.T) {
 	if err := tc.SetupPlaywright(withEnv); err != nil {
 		t.Fatalf("fail to setup playwright: %v", err)
 	}
-	if stdout, stderr, err := tc.RunPlaywrightTest("deploy-ec23-app"); err != nil {
-		t.Fatalf("fail to run playwright test deploy-ec23-app: %v: %s: %s", err, stdout, stderr)
-	}
-
-	// TODO (@salah): use shared join function once we release 2.4.0 as it becomes the "previous stable"
```

Comment:
there's another TODO above that should be removed

Reply:
removed